src/Catalyst.Core.Lib/Kernel/Kernel.cs

```diff
@@ -81,6 +82,7 @@ private Kernel(ICancellationTokenProvider cancellationTokenProvider, string file
             Logger = new LoggerConfiguration()
                .WriteTo.Console()
                .WriteTo.File(Path.Combine(Path.GetTempPath(), fileName), rollingInterval: RollingInterval.Day)
+               .Filter.ByExcluding(Matching.FromSource("Microsoft"))
```

Comment:
do you want to keep it? it removes all the messy ASP logging

Reply:
Yes I think so we can put some separate logging for the Web3 API stuff